todo-list-ui/src/app/HelloWorld/helloWorld.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from "@angular/core";
+
+@Component({
+    selector: "app-helloWorld",
```

Comment:
Use dashed-case when naming component selectors: https://angular.dev/style-guide#style-05-02

Also consider the Todo component comment here as well.